bin/import-borders

```diff
@@ -4,10 +4,10 @@ set -o pipefail
 set -o nounset
 
 : "${IMPORT_DIR:=/import}"
-# Do not use .pbf extension to avoid accidental conflict with the original pbf with multiple runs
+: "${BORDERS_CLEANUP:=false}"
+: "${BORDERS_CLEANUP_FILE:="$IMPORT_DIR/borders/cleanup.pbf"}"
+: "${BORDERS_CLEANUP_BBOX:=-180,-90,180,90}"
```

Comment:
the cleanup orders should not change as this is just a trigger to activate --drop-broken-refs. It is set to the maximum in order to avoid any extra computation in osmconvert.